src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -114,100 +137,244 @@ object VCFOutput {
 
     def allelicDepthString(depths: Map[String, Int]): String = {
       val allAlleles = (Seq(allele.ref, allele.alt) ++ depths.keys.toSeq).distinct
-      allAlleles.map(x => "%s=%d".format(if (x.isEmpty) "." else x, depths.getOrElse(x, 0))).mkString(",")
+      allAlleles.map(
```

Comment:
Do you mean that specifically about the change at this line, or this whole PR?

I agree this line is a little more vertically stretched than it needs to be; will condense it back a bit.